cohere/client.py

```diff
@@ -208,7 +208,7 @@ def classify(self,
         return Classifications(classifications)
 
     def summarize(self, text: str, model: str = None, length: str = None, format: str = None, temperature: float = None,
-                  additional_instruction: str = None) -> SummarizeResponse:
+                  additional_instruction: str = None, abstractiveness: str = None) -> SummarizeResponse:
```

Comment:
is this getting added to the json?

Reply:
![bitmoji](https://sdk.bitmoji.com/render/panel/01c81339-ddd6-46df-b141-ba3c6f2bcad3-d9287ff3-ffc3-44fa-aaa7-b63f115c679f-v1.png?transparent=1&palette=1&width=246)